Gulpfile.ts

```diff
@@ -434,55 +435,50 @@ const nodePackageFile = path.join(builtLocalDirectory, "typescript.js");
 const nodeDefinitionsFile = path.join(builtLocalDirectory, "typescript.d.ts");
 const nodeStandaloneDefinitionsFile = path.join(builtLocalDirectory, "typescript_standalone.d.ts");
 
+const prependCopyright = lazypipe()
+    .pipe(() => insert.prepend(fs.readFileSync(copyright)));
+
 gulp.task(builtLocalCompiler, false, [servicesFile], () => {
     const localCompilerProject = tsc.createProject("src/compiler/tsconfig.json", getCompilerSettings({}, /*useBuiltCompiler*/true));
     return localCompilerProject.src()
         .pipe(newer(builtLocalCompiler))
         .pipe(sourcemaps.init())
         .pipe(tsc(localCompilerProject))
-        .pipe(gIf(useDebugMode, insert.prepend(fs.readFileSync(copyright))))
+        .pipe(gIf(useDebugMode, prependCopyright()))

```

Comment:
Or why not just `insert.prepend(useDebugMode ? fs.readFileSync(copyright) : "")`, any avoid both of those dependencies?